pandas/core/style.py

```diff
@@ -255,6 +272,75 @@ def _translate(self):
                     precision=precision, table_styles=table_styles,
                     caption=caption, table_attributes=self.table_attributes)
 
+    def format(self, formatter, subset=None):
+        '''
+        Format the text display value of cells.
+
+        .. versionadded:: 0.18.0
+
```

Comment:
Another option could be a `select_dtypes` like method that only returns the columns with the specified dtypes: i.e. give the they numeric column names. Much of the time I use `select_dtypes`, I'm using it as an indexer into the columns, `df.columns[df.select_dtypes(['numeric'])].columns]`. This would be a bit more typing in this specific case, but broadly useful.